.vscode/extensions.json

```diff
@@ -0,0 +1,9 @@
+{
+	"recommendations": [
+		"streetsidesoftware.code-spell-checker",
+		"streetsidesoftware.code-spell-checker-portuguese-brazilian",
+		"editorconfig.editorconfig",
+		"dbaeumer.vscode-eslint",
+		"eamodio.gitlens"
```

Comment:
Should we add `esbenp.prettier-vscode` too?

Reply:
I think so...

Reply:
It might conflict with the ESLint one.